GhostNetwork.Gateway.Api/NewsFeedController.cs

```diff
@@ -31,6 +31,36 @@ public async Task<ActionResult<NewsFeedPublication>> CreateAsync([FromBody] Crea
         {
             return Ok(await source.CreateAsync(model.Content));
         }
+
+        [HttpPut("{id}")]
+        [ProducesResponseType(StatusCodes.Status204NoContent)]
+        [ProducesResponseType(StatusCodes.Status404NotFound)]
```

Comment:
Use 204 for successful publication updating, 404 only when publication does not exist and 400 when validation failed